ert_shared/dark_storage/endpoints/records.py

```diff
@@ -157,8 +175,8 @@ async def get_ensemble_record(
 ) -> Any:
     ensemble_name = get_name("ensemble", ensemble_id)
     dataframe = data_for_key(ensemble_name, name)
-    if realization_index:
-        dataframe = dataframe[realization_index]
+    if realization_index or realization_index == 0:
+        dataframe = pd.DataFrame(dataframe.loc[realization_index]).T
```

Comment:
Maybe add a comment explaining what is going on. I don't understand why you need this transformation